modules/lofreq/1.1/lofreq.smk

```diff
@@ -82,6 +83,25 @@ def _lofreq_get_capspace(wildcards):
 
 ##### RULES #####
 
+# Setup LoFreq to use system-specific binaries, but custom python scripts
+# By default, the LoFreq binary (from my understanding) uses python scripts in the same directory as the binary
+# So lets put the conda binary with our python scripts
+rule _lofreq_configure_software:
```

Comment:
It won't fall back to the default binaries, since the python scripts packaged with LoFreq don't have some of the modifications we've made (ex. the --normal-only). As it appears the LoFreq binary appears to use the python scripts located in the same directory as the LoFreq binary by default (from my understanding, could be wrong), the "general" binary has to be copied to the same location as these modified scripts

Reply:
P.S. the "default binary" portion wouldn't work, as it would just print that message (WARNING: PATH is not set properly, using $(which lofreq2_call_pparallel.py)") and then "complete" the rule

Reply:
Ah, I see - you are right! There was nothing after the message, so the "default" lofreq will not run anyways. Can you at least please include the echo statement which will print the $PATH and $SCRIPT for future debugging purposes?